### ISSUE TITLE
Unknown version 73 of firefox

### ISSUE THREAD

**Issue by equinusocio** [browserslist/browserslist#441]
Some recent Browserslist updates broke my postcss build process (using parcel and postcss-preset-env). The Firefox query return an error about missing version 73.

```js
  "browserslist": [
    "last 1 Chrome versions",
    "last 1 Firefox versions", <--- broken
    "last 1 Safari versions",
    "last 1 Edge versions"
  ]
```

```bash
./ne-core.pcss: Unknown version 73 of firefox
    at Function.select (test/node_modules/browserslist/index.js:1001:17)
    at test/node_modules/browserslist/index.js:277:33
    at Array.reduce (<anonymous>)
    at resolve (test s/node_modules/browserslist/index.js:259:18)
    at browserslist (test/node_modules/browserslist/index.js:383:21)
    at getTargets test/node_modules/@babel/helper-compilation-targets/lib/index.js:201:48)
    at _default (test/node_modules/@babel/preset-env/lib/index.js:228:57)
    at Object.default (test/node_modules/@babel/helper-plugin-utils/lib/index.js:19:12)
    at getEnvPlugins (test/node_modules/parcel-bundler/src/transforms/babel/env.js:62:34)
    at getEnvConfig (test/node_modules/parcel-bundler/src/transforms/babel/env.js:12:25)
```

Using npx

```bash
➜ npx browserslist "last 1 Chrome versions, last 1 Firefox versions, last 1 Safari versions, last 1 Edge versions"
chrome 80
edge 76
firefox 73
safari TP
```

**Comment by ai** [comment]
Seems like https://github.com/browserslist/browserslist/pull/438 fixes it too (thanks for the issue anyway, the question was why do you have `unknown browser`, seems like Babel uses as in wrong way)

**Comment by ai** [comment]
Fixed at 4.8.5

**Comment by equinusocio** [comment]
Thanks! So fast!
